src/main/java/com/google/api/gax/grpc/BundlingSettings.java

```diff
@@ -103,6 +103,12 @@
   /** Returns the Boolean object to indicate if the bundling is enabled. Default to true */
   public abstract Boolean getIsEnabled();
 
+  public abstract Boolean getIsFlowControlEnabled();
```

Comment:
I don't think we need this method. If we don't want to enable flow control, just set the fields of `FlowControlSettings` to null. When initialized in this way, the `FlowController` methods are noop. Then we don't need to check if the controller is null or not.